pipeline/data/dataset_importer.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+"""
+Downloads a dataset and runs augmentation if needed
+
+Example:
+    SRC=ru TRG=en python pipeline/data/dataset_importer.py \
+        --type=corpus \
+        --dataset=sacrebleu_aug-mix_wmt19 \
+        --output_prefix=$(pwd)/test_data/augtest
+"""
+
+import argparse
+import os
+import re
+import subprocess
+import sys
+from typing import Iterable, List
+
+from opustrainer.modifiers.surface import TitleCaseModifier, UpperCaseModifier
+from opustrainer.modifiers.typos import TypoModifier
+from opustrainer.types import Modifier
+
+
+class CompositeModifier:
+    def __init__(self, modifiers: List[Modifier]):
+        self._modifiers = modifiers
+
+    def __call__(self, batch: List[str]) -> Iterable[str]:
+        for mod in self._modifiers:
+            batch = list(mod(batch))
+
+        return batch
+
+
+MIX_PROB = 0.1  # 10% will be augmented in the mix
```

Comment:
Would it make sense to pass this in?

Reply:
it's fine to leave it as a constant, we'll not change this often, maybe never